src/components/select/select.spec.js

```diff
@@ -153,6 +153,22 @@ describe('<md-select>', function() {
     expect($rootScope.myForm.select.$touched).toBe(true);
   }));
 
+  it('should remain untouched during opening', inject(function($compile, $rootScope) {
+    var form = $compile('<form name="myForm">' +
+                        '<md-select name="select" ng-model="val">' +
+                        '<md-option>1</md-option>' +
+                        '</md-select>' +
+                        '</form>')($rootScope);
+    var unwatch = $rootScope.$watch('myForm.select.$touched',
```

Comment:
Can you explain why this is in a `$watch` expression instead of just calling `$rootScope.$eval('myForm.select.$touched')` within the test code below?